ql/src/test/results/clientpositive/llap/materialized_view_partitioned.q.out

```diff
@@ -1154,11 +1154,11 @@ POSTHOOK: Input: default@partition_mv_3
 POSTHOOK: Input: default@partition_mv_3@key=238
 #### A masked pattern was here ####
 val_238_n	238
-val_238_n	238
 val_238	238
 val_238	238
 val_238	238
 val_238	238
+val_238_n	238
```

Comment:
Although this doesn't violate the semantics, I didn't understand why this diff showed up

Reply:
Since the query
```
SELECT * FROM partition_mv_3 where key = 238
```
does not specify ordering I think it depends on the execution engine internals.